### PR TITLE
Fix bug with no modifier inside element.

### PR DESCRIPTION
**Pull request by Yeti-or**
https://github.com/bem/webpack-bem-loader/issues/14

### CODE REVIEW

**Comment by awinogradov** [comment]
Close #2

File: index.js [hunk 1]
```diff
@@ -14,65 +14,76 @@
...
-    });
 
-    return res;
+        return acc;
+    }, []);
 }
 
-module.exports = parseImport;
+module.exports = {
+    matchRegExp : /^(b|e|m|t)\:/,
```
Comment (Yeti-or) [inline index.js]: remove

**Comment by Yeti-or** [comment]
bem-import-string
bem-import-util

File: index.js [hunk 2]
```diff
@@ -14,65 +14,76 @@
...
+
+            modVals && modVals.split('|').forEach(modVal => {
+                acc.push(Object.assign({}, main, { mod : { name : modName, val : modVal } }));
+            });
+            break;
+
+        case 't':
+            main.elem || main.block || (main.elem = ctx.elem);
+            main.block || (main.block = ctx.block);
+            acc = (acc.length ? acc : [main]).map(e => {
```
Comment (Yeti-or) [inline index.js]: acc.push(main)
acc.forEach(e => (e.tech = t))

File: README.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+[test-img]:       https://img.shields.io/travis/bem/bem-import.svg?label=tests
+
+[coveralls]:      https://coveralls.io/r/bem/bem-import
+[coverage-img]:   https://img.shields.io/coveralls/bem/bem-import.svg
+
+[david]:          https://david-dm.org/bem/bem-import
+[dependency-img]: http://img.shields.io/david/bem/bem-import.svg
+
+
+Parse your `import 'b:button2'`, yo!
```
Comment (qfox) [inline README.md:22]: Parses

File: README.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
+bem-import
+==========
+
```
Comment (qfox) [inline README.md:3]: Need to put description here for npm

**Pull request merged**